release_notes/47-release-notes.adoc

```diff
@@ -478,4 +480,30 @@ This release provides fixes for the following security issues:
 * zlib: Out-of-bounds pointer arithmetic in `inftrees.c` link:https://access.redhat.com/security/cve/CVE-2016-9840[(CVE-2016-9840)]
 * krb5: Kerberos RC4-HMAC-MD5 checksum vulnerability link:https://access.redhat.com/security/cve/CVE-2025-3576[(CVE-2025-3576)]
 
+[id="about-release-475_{context}"]
+== About release version 4.7.5
+
+*Release date*: 15 July 2025
+
+This release of {product-title-short} 4.7 provides the following bug fixes:
+
+//ROX-29320
+* Before this update, incorrect interpretation of Red Hat Enterprise Linux (RHEL) 10 Common Platform Enumeration (CPE) strings caused Scanner V4 to fail distribution checks on RHEL 10 systems. With this update, an updated RHEL CPE major version pattern resolves the issue, and Scanner V4 can now correctly support RHEL 10.
+
+//ROX-29755
+* Before this update, the failure of Sensor to call stream.Recv() caused gRPC flow control to block image reprocessing every 4 hours. With this update, the reprocessing loop includes a timeout for sending messages to Sensors, which resolves the issue and and resumes the image reprocessing as expected.
+
+//ROX-28665
+* Before this update, missing network policy configuration for port `9090` on long-running clusters from earlier releases prevented Elastic search from indexing those clusters. With this update, Elastic search integration for long-running cluster metrics resolves the issue, and you can now access search results from these clusters.
+
+//ROX-29649
+* Before this update, repeated access to the Singleton instance caused excessive logging of telemetry collection status, resulting in log spam over a 5 day period. With this update, removal of the telemetry collection log from the `InstanceConfig` function resolves the issue and reduces unnecessary log entries.
```

Comment:
I think, we share too many code-related details that are irrelevant to the users.
```suggestion
* Before this update,  one could observe excessive logging of telemetry collection status, resulting in log spam. With this update, the telemetry collection has been configured to not emit repeated logs continuously, which significantly reduces the log volume.
```